tests/tls/tls-subjectaltname/suricata.yaml

```diff
@@ -0,0 +1,17 @@
+%YAML 1.1
+---
+
+outputs:
+  - eve-log:
+      enabled: yes
+      filetype: regular #regular|syslog|unix_dgram|unix_stream|redis
+      filename: eve.json
+      types:
+        - alert:
+            payload: no
+            payload-buffer-size: 4kb
+            payload-printable: no
+            packet: no
+            metadata: no
+        - tls:
+            custom: [subject, issuer, serial, fingerprint, sni, version, not_before, not_after, subjectaltname]
```

Comment:
added this to ensure that we're logging `subjectaltname` using the custom logging function.